help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+
```

Comment:
we have several sections missing here, before we can get to the **Connect to the destination** part. 

we need a title for the page, an overview, use cases, prerequisites, etc, as per the self-service documentation template: https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en

see also for example the documentation by Amazon for their Amazon Ads destination: https://github.com/AdobeDocs/experience-platform.en/edit/main/help/destinations/catalog/advertising/amazon-ads.md